examples/java/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ plugins {
 
 dependencies {
     implementation(projects.xefJava)
+    implementation(libs.kotlinx.coroutines.reactive)
```

Comment:
Since the `promptStreaming : Flow<String>` support is available in xef-core this dependency should appear as part of the `xef-java` example instead of the examples. `xef-java` is the one providing `promptStreaming: Publisher<String>`

Reply:
Done